cmake/projects/OpenSSL/hunter.cmake

```diff
@@ -88,6 +88,28 @@ hunter_add_version(
     fb3bc6467d37f7536f5ff7a1ea25552cee899275
 )
 
+hunter_add_version(
+    PACKAGE_NAME
+    OpenSSL
+    VERSION
+    "1.1.1h"
+    URL
+    "https://www.openssl.org/source/old/1.1.1/openssl-1.1.1h.tar.gz"
+    SHA1
+    8d0d099e8973ec851368c8c775e05e1eadca1794
+)
+
+hunter_add_version(
+    PACKAGE_NAME
+    OpenSSL
+    VERSION
+    "1.1.1i"
+    URL
+    "https://www.openssl.org/source/openssl-1.1.1i.tar.gz"
```

Comment:
Can you use the Github release instead? These links will break when a new version comes out. https://github.com/openssl/openssl/archive/OpenSSL_1_1_1i.tar.gz for example.